test/backend_tests/test_resolve_fetch_failures.py

```diff
@@ -0,0 +1,32 @@
+"""Tests
+
+How to run:
+    python -m unittest discover
+"""
+import os
+import sys
+from pathlib import Path
+
+from test.backend_tests.db.test_utils import FetchAuditTestRunner
+
+TEST_DIR = os.path.dirname(__file__)
+PROJECT_ROOT = Path(TEST_DIR).parent.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+
+
+class TestBackendResolveFetchFailures(FetchAuditTestRunner):
+    """Inherits from the FetchAudit test class to get setup/teardown methods."""
+    # TODO: Need a better test. This one actually passes even though lots of members. i need one that has more than 10k
+    #  items
+    # Hope Termhub Test (v3) https://unite.nih.gov/workspace/module/view/latest/ri.workshop.main.module.5a6c64c0-e82b-4cf8-ba5b-645cd77a1dbf
+    mock_data = [{'table': 'code_sets', 'primary_key': 946809371, 'status_initially': 'fail-excessive-items',
```

Comment:
@Sigfried Just wanna show you something strange. I was trying to come up with a case where something was actually failing. I found this case:

LongCovid (v1) (`112182256`)
```
{
   "properties": {
     "updateMessage": "Long Covid features",
     "lastModifiedAt": "2023-06-27T01:24:03.447Z",
     "sourceApplicationVersion": "2.0",
     "lastModifiedBy": "3b4fee14-4577-4fb3-98aa-f25e14fb0a18",
     "createdBy": "3b4fee14-4577-4fb3-98aa-f25e14fb0a18",
     "version": 1,
     "createdAt": "2023-06-27T00:11:50.735Z",
     "project": "RP-92E57A",
     "sourceApplication": "UNITE",
     "provenance": "Long Covid features",
     "conceptSetNameOMOP": "LongCovid",
     "limitations": "included from paper",
     "intention": "Narrow (specific)",
     "conceptSetVersionTitle": "LongCovid (v1)",
     "codesetId": 112182256,
     "isDraft": false,
     "isMostRecentVersion": true
   },
   "rid": 
 "ri.phonograph2-objects.main.object.241458e2-d61f-4f9a-9409-b7e0b6b96a68"
 }
```

However, when I look it up in the enclave, [nothing comes up](https://unite.nih.gov/workspace/module/view/latest/ri.workshop.main.module.5a6c64c0-e82b-4cf8-ba5b-645cd77a1dbf):
> No Concept Sets found matching the Search and Filtering Criteria

And when I try to fetch its container to see if it's archived, I don't see a container either. This returns an empty list:
```py
    make_objects_request('OMOPConceptSetContainer', query_params={'properties.conceptSetId.eq': _id}, verbose=verbose, return_type='data')
```

Just curious if you have any idea why that might be?

Reply:
@Sigfried I ran a fetch since January 1st and so far this one above is actually the only failure case. Going to run again to be sure.